dart/lib/src/event_processor/enricher/io_enricher_event_processor.dart

```diff
@@ -12,9 +12,26 @@ EnricherEventProcessor enricherEventProcessor(SentryOptions options) {
 /// Uses Darts [Platform](https://api.dart.dev/stable/dart-io/Platform-class.html)
 /// class to read information.
 class IoEnricherEventProcessor implements EnricherEventProcessor {
-  IoEnricherEventProcessor(this._options);
+  IoEnricherEventProcessor(this._options) {
+    dartVersion = _extractDartVersion(Platform.version);
+  }
 
   final SentryOptions _options;
+  late final String dartVersion;
```

Comment:
how about automatic init when needed? 
```suggestion
  IoEnricherEventProcessor(this._options);
  
  final SentryOptions _options;
  late final String dartVersion = _extractDartVersion(Platform.version);
```

Reply:
and maybe don't expose it as it's just an implementation detail